<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionShowLineNumber"/>
    <addaction name="actionUneWrap"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionFontColor"/>
    <addaction name="separator"/>
    <addaction name="actionFontBackground"/>
    <addaction name="actionEditorBackground"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionShowToolbar"/>
    <addaction name="actionShowstatusbar"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>书签(&amp;B)</string>
    </property>
    <addaction name="actionAddBookmark"/>
    <addaction name="actionRemoveBookmark"/>
    <addaction name="actionViewBookmarks"/>
    <addaction name="actionGotoBookmark"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
   <addaction name="menuBookmarks"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionShowLineNumber"/>
   <addaction name="actionUneWrap"/>
   <addaction name="separator"/>
   <addaction name="actionShowToolbar"/>
   <addaction name="actionShowstatusbar"/>
   <addaction name="actionToggleTheme"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew" resource="img.qrc">
     <normaloff>:/img/new2.bmp</normaloff>
     <normalon>:/img/120.bmp</normalon>:/img/new2.bmp</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="toolTip">
    <string>新建(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut" resource="img.qrc">
     <normaloff>:/img/cut.bmp</normaloff>:/img/cut.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切(X)</string>
   </property>
   <property name="toolTip">
    <string>剪切(X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemLogOut" resource="img.qrc">
     <normaloff>:/img/Exit.png</normaloff>:/img/Exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave" resource="img.qrc">
     <normaloff>:/img/Save.png</normaloff>:/img/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs" resource="img.qrc">
     <normaloff>:/img/Saveas.png</normaloff>:/img/Saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
   <property name="text">
    <string>撤销(Z)</string>
   </property>
   <property name="toolTip">
    <string>撤销(Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="toolTip">
    <string>打开(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="toolTip">
    <string>复制(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste"/>
   </property>
   <property name="text">
    <string>粘贴(V)</string>
   </property>
   <property name="toolTip">
    <string>粘贴(V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
   </property>
   <property name="text">
    <string>查找(F)</string>
   </property>
   <property name="toolTip">
    <string>查找(F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaylistRepeat"/>
   </property>
   <property name="text">
    <string>替换(H)</string>
   </property>
   <property name="toolTip">
    <string>替换(H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextItalic"/>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
   <property name="toolTip">
    <string>字体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionUneWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyCenter"/>
   </property>
   <property name="text">
    <string>自动换行</string>
   </property>
   <property name="toolTip">
    <string>自动换行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionShowToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/IcOutlineHandyman.png</normaloff>:/img/IcOutlineHandyman.png</iconset>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
   <property name="toolTip">
    <string>工具栏</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionShowstatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DriveHarddisk"/>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
   <property name="toolTip">
    <string>状态栏</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>重做(shift+Z)</string>
   </property>
   <property name="toolTip">
    <string>重做(shift+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
   </property>
   <property name="text">
    <string>全选(A)</string>
   </property>
   <property name="toolTip">
    <string>全选(A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionShowLineNumber">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove"/>
   </property>
   <property name="text">
    <string>显示行号</string>
   </property>
   <property name="toolTip">
    <string>显示行号</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="toolTip">
    <string>字体颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEditorBackground">
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="toolTip">
    <string>编辑器背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontBackground">
   <property name="text">
    <string>字体背景色</string>
   </property>
   <property name="toolTip">
    <string>字体背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToggleTheme">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
   </property>
   <property name="text">
    <string>切换深浅色模式</string>
   </property>
   <property name="toolTip">
    <string>切换为深/浅色模式</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddBookmark">
   <property name="text">
    <string>添加书签</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRemoveBookmark">
   <property name="text">
    <string>移除书签</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionViewBookmarks">
   <property name="text">
    <string>查看书签</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionGotoBookmark">
   <property name="text">
    <string>跳转书签</string>
   </property>
   <property name="toolTip">
    <string>跳转到指定书签</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
